id/news/_posts/2019-08-28-ruby-2-4-7-released.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "Ruby 2.4.7 Rilis"
+author: "usa"
+translator: "meisyal"
+date: 2019-08-28 09:00:00 +0000
+lang: id
+---
+
+Ruby 2.4.7 telah dirilis.
+
+Rilis ini mencakup sebuah perbaikan keamanan.
+Mohon cek topik-topik di bawah ini untuk detail.
+
+* [Beberapa kerentanan jQuery pada RDoc](/id/news/2019/08/28/multiple-jquery-vulnerabilities-in-rdoc/)
+
+Ruby 2.4 saat ini berada pada fase perawatan keamanan hingga akhir Maret 2020.
```

Comment:
I think "pemeliharaan" is better than "perawatan"

Reply:
This one also. Could you check it again, @kuntoaji?